dts/imx8mm-cgtsx8m-ultimain5.0-lvds-1024x600.dts

```diff
@@ -150,6 +150,36 @@
             linux,default-trigger = "default-on";
         };
     };
+
```

Comment:
Yeah, unfortunately - **as far as I know** - there is no way to check if a device is preset. The Device Tree is basically a hardware description to the kernel. At this point, the device tree tells the kernel to initialize 2 nodes, one named `cabin_light` at the address `0x61` and another named `calib_light_2` at address `0x62`. Since we know that only one of these devices will be present, the other is expected to fail.